src/components/fx/hover.js

```diff
@@ -153,6 +153,82 @@ exports.loneHover = function loneHover(hoverItem, opts) {
     return hoverLabel.node();
 };
 
+// TODO: replace loneHover?
```

Comment:
Interesting... can you say more? Is this just an extension of `loneHover` to support multiple labels or does it add something else? I only see one label at a time when I play with parcats.

Reply:
At one point I had added a mode that would display a separate tooltip for each color of the category you hover over.  It got pretty unwieldy and I reverted back to `loneHover`.  I'll remove this `customHover` function.

Reply:
Unwieldy in the code or on screen? We have some upcoming `sankey` enhancements that may call for a very similar multi-label hover effect, though the details still need to be worked out. But if the code is clean and it was just not looking good in practice, it still may be worth keeping this around.

Reply:
On the screen. I think the implementation worked well enough. I'll look over it again and write a better explanation of what it does 🙂